src/test/java/org/takes/rs/RsWithTypeTest.java

```diff
@@ -52,11 +57,41 @@ public void replaceTypeToResponse() throws IOException {
                 )
             ).print(),
             Matchers.equalTo(
-                Joiner.on("\r\n").join(
-                    "HTTP/1.1 200 OK",
-                    String.format("Content-Type: %s", type),
+                Joiner.on(CRLF).join(
+                        "HTTP/1.1 200 OK",
+                        String.format("Content-Type: %s", type),
+                        "",
+                        ""
+                )
+            )
+        );
+    }
+
+    /**
+     * RsWithType can not replace response code.
+     * @throws java.io.IOException If a problem occurs.
+     */
+    @Test
+    public void notReplacesResponseCode() throws IOException {
+        final int code = 500;
+        final String body = "Error!";
+        MatcherAssert.assertThat(
+            new RsPrint(
+                new RsWithType(
+                    new RsWithBody(
+                        new RsWithStatus(code),
```

Comment:
@ekondrashev We should use constant `HttpURLConnection.HTTP_INTERNAL_ERROR` [(see javadoc)](http://docs.oracle.com/javase/7/docs/api/java/net/HttpURLConnection.html#HTTP_INTERNAL_ERROR).